test/integration/go.mod

```diff
@@ -47,10 +47,10 @@ require (
 	github.com/cespare/xxhash/v2 v2.2.0 // indirect
 	github.com/chai2010/gettext-go v1.0.2 // indirect
 	github.com/containerd/containerd v1.7.0 // indirect
```

Comment:
Upgrade containerd to v1.7.1 to address 2 critical & 5 high vulnerabilities.
CVE-2023-24540, CVE-2023-24538, CVE-2023-24536, CVE-2023-24537, CVE-2023-24539, CVE-2023-29400 & CVE-2023-24534